server/src/main/java/org/apache/druid/rpc/DiscoveryServiceLocator.java

```diff
@@ -157,5 +157,18 @@ public void nodeViewInitialized()
         }
       }
     }
+
+    @Override
+    public void nodeViewInitializedTimedOut()
+    {
+      synchronized (DiscoveryServiceLocator.this) {
```

Comment:
Since it's the same as `nodeViewInitialized()`, this should just call `nodeViewInitialized();`

Reply:
Done